### PR TITLE
Review: Requirements

### LINKED ISSUE #243: Review: Requirements

### PR DESCRIPTION
**Pull request by Kechr**

### CODE REVIEW

**Comment by Joergenson** [comment]
Line 8: A summary of is made
Line 93:  Is there meant to be 2x S14?

**Comment by Kechr** [comment]
L002: emphasises on
L002: comma after customer
L008: delete 'of' -> maybe rewrite the part
L011: delete this part, no need to conclude
L014: No need to write the full name, MES is fine as we've written it before
L019: batch number - batch id (?)
L019: "After a finished ..." -> rewrite this sentence
L024: remove 'this'
L037: Not the system.
L038: Not the system.
L045: no need to write C and W, as they are not used.
L064: batch number -> batch id (?)
L089: Remove personal pronoun
L109: No design constraints?
L113: What should?
L143: Rewrite to include requirement -> right now there's no connection
L174: Fix spacing
L183: Control panel -> dashboard
L215: Clarify -> start the machine
L218: More preconditions: Maintenence, ingredients.
L226: The user doesn't set the batch id.
L234: Clarify ->  production has started
L246: Write a more describing intro to this section. What are the green bubbles?

**Comment by kkmp-dk** [comment]
L018 new machine -> maybe just machine?
L085 to the system to be developed -> to the developed system? 
L121 table Supplementary Requirements S12 og S13 to be together? So it is a part of the batch report
L169 In order to determine who will be able to access the system, the actors are found -> maybe rephrase?

**Comment by Bokaiii** [comment]
Requirements:
- 3.2 Selected detailed requirements
    - Which “describes” = describe
